.gitignore

```diff
@@ -20,10 +20,13 @@ var/*
 .tox
 .DS_Store
 .coverage
+.coverage.*.*
 .noseids
 docs/_build/
 nosetests.xml
 __pycache__/
-htmlcov
+htmlcov/
+_dynamic/
```

Comment:
A directory to store dumps of possibly-useful info when running tox's testenvs (e.g., a dump of the currently-seen env vars)

Reply:
👍